tests/test_api.py

```diff
@@ -1,5 +1,15 @@
+import asyncio
```

Comment:
We may need add a bit test for heartbeat message is not displayed to users

Reply:
Good catch, added!